packages/web-react/src/components/SplitButton/stories/UncontrolledSplitButton.stories.tsx

```diff
@@ -0,0 +1,142 @@
+    },
+    color: {
+      control: 'select',
+      options: [...Object.values(ComponentButtonColors).filter((color) => color !== ComponentButtonColors.PLAIN)],
+      description: 'Color of the button.',
+      table: {
+        defaultValue: { summary: ComponentButtonColors.PRIMARY },
+        type: { summary: 'ButtonColor' },
+      },
+    },
+    buttonLabel: {
+      control: 'text',
+      description: 'The label for the button.',
+      table: {
+        type: { summary: 'string' },
+      },
+    },
+    buttonIconName: {
+      control: 'text',
+      description: 'The name of the icon to display on the button.',
+      table: {
+        type: { summary: 'string' },
+      },
+    },
+    dropdownLabel: {
+      control: 'text',
+      description: 'The label for the dropdown button.',
+      table: {
+        type: { summary: 'string' },
+      },
+    },
+    children: {
+      control: 'text',
+      description: 'The content of the dropdown.',
+      table: {
+        type: { summary: 'ReactNode' },
+      },
+    },
+    dropdownPlacement: {
+      control: 'select',
+      options: Object.values(Placements),
+      table: {
+        defaultValue: { summary: Placements.BOTTOM_END },
+      },
+      description: 'The placement of the dropdown.',
+    },
+    isDisabled: {
+      control: 'boolean',
+      description: 'Whether the button is disabled.',
+      table: {
+        type: { summary: 'boolean' },
+      },
+    },
+    dropdownIconName: {
+      control: 'text',
+      description: 'The name of the icon to display on the dropdown button.',
+      table: {
+        type: { summary: 'string' },
+      },
+    },
+    buttonOnClick: {
+      description: 'Function to call when the button is clicked.',
+      table: {
+        type: { summary: '() => void' },
+      },
+    },
+    isLabelHidden: {
+      control: 'boolean',
+      description: 'Whether the button label is hidden.',
+      table: {
+        type: { summary: 'boolean' },
+      },
+    },
+    isDropdownLabelHidden: {
+      control: 'boolean',
+      description: 'Whether the dropdown label is hidden.',
+      table: {
+        type: { summary: 'boolean' },
+      },
+    },
+  },
+  args: {
+    buttonIconName: undefined,
+    buttonLabel: 'Button',
+    buttonOnClick: () => {},
+    children: <DropdownContent />,
+    color: ComponentButtonColors.PRIMARY,
+    dropdownIconName: 'chevron-down',
+    dropdownLabel: 'More',
+    dropdownPlacement: Placements.BOTTOM_END,
+    id: 'uncontrolled-split-button',
+    isDisabled: false,
+    isLabelHidden: false,
+    isDropdownLabelHidden: false,
+    size: Sizes.MEDIUM,
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof UncontrolledSplitButton>;
+
+export const UncontrolledSplitButtonPlayground: Story = {
+  name: 'UncontrolledSplitButton',
+  render: (args) => {
+    const { children, ...rest } = args;
+
+    return <UncontrolledSplitButton {...rest}>{children}</UncontrolledSplitButton>;
+  },
+};
```

Comment:
```suggestion
export const UncontrolledSplitButtonPlayground: Story = {
  name: 'UncontrolledSplitButton',
};
```

this is not enough? :O

Reply:
it is!